Ports/libopenal/patches/isfinite.patch

```diff
@@ -0,0 +1,182 @@
+diff -Naur openal-soft-1.21.1/al/source.cpp openal-soft-1.21.1.serenity/al/source.cpp
+--- openal-soft-1.21.1/al/source.cpp	2021-02-04 20:09:06.000000000 +0100
++++ openal-soft-1.21.1.serenity/al/source.cpp	2021-05-08 05:31:29.290325847 +0200
+@@ -1232,14 +1232,14 @@
+ 
+     case AL_SOURCE_RADIUS:
+         CHECKSIZE(values, 1);
+-        CHECKVAL(values[0] >= 0.0f && std::isfinite(values[0]));
++        CHECKVAL(values[0] >= 0.0f && __builtin_isfinite(values[0]));
```

Comment:
`isfinite` exists in `math.h` as a standalone function, might be able to get away with that instead of the intrinsic